x-pack/plugins/osquery/cypress/tasks/live_query.ts

```diff
@@ -11,12 +11,14 @@ export const DEFAULT_QUERY = 'select * from processes;';
 export const BIG_QUERY = 'select * from processes, users limit 110;';
 
 export const selectAllAgents = () => {
-  cy.react('AgentsTable').find('input').should('not.be.disabled');
```

Comment:
I refactor this to stop using cy.react because not everywhere cypress react selector is available (event details flyout).